Sources/App/routes.swift

```diff
@@ -28,7 +30,25 @@ func routes(_ app: Application) throws {
         }
     }
     
-    app.routes.get("admin") { request -> View in
+    app.get("conduct") { req -> EventLoopFuture<View> in
+        return req.view.render("Secondary/conduct", ["name": "Leaf"])
+    }
+
+    try app.routes.register(collection: AuthController()) // TODO: Split this out into web/api/admin
+    try app.routes.register(collection: SpeakerController()) // TODO: Split this out into web/api/admin
+    try app.routes.register(collection: EventsController()) // TODO: Split this out into web/api/admin
+    
+    // MARK: - API Routes
+    
+    let apiRoutes = app.grouped("api", "v1")
+    
+    try apiRoutes.grouped("presentations").register(collection: PresentationAPIController())
```

Comment:
Looks like we're using update routes for the api namespace, I'd have thought this would be admin routes, keeping the api routes for the actual api used for the app?

Reply:
Just thought - if the above is the case to move it under admin, might be worth a whole folder structure shuffle to put `Features/Presentations/...` under `Admin/` and the same for speakers, etc?

Reply:
I'd say they're nothing to do with an api at all so should just be admin routes for updating content.  That's how I did the CodeMobile backend

Reply:
Yeah, that's how I split things up, for views and other files

Reply:
I also used a different base, css and js for admin as mine was totally different, so you don't have to bleed any non-admin code to the main site